terraform/datadog/aws.tf

```diff
@@ -1,7 +1,7 @@
 resource "datadog_monitor" "ec2-cpu-credits" {
   name           = "${format("%s EC2 CPU credits", var.env)}"
   type           = "query alert"
-  message        = "${format("Instance is {{#is_warning}}low on{{/is_warning}}{{#is_alert}}out of{{/is_alert}} on CPU credits and may perform badly. See: https://government-paas-team-manual.readthedocs.io/en/latest/support/responding_to_alerts/#cpu-credits @govpaas-alerting-%s@digital.cabinet-office.gov.uk", var.aws_account)}"
+  message        = "${format("Instance is {{#is_warning}}low on{{/is_warning}}{{#is_alert}}out of{{/is_alert}} CPU credits and may perform badly. See: %s#cpu-credits @govpaas-alerting-%s@digital.cabinet-office.gov.uk", var.datadog_documentation_url, var.aws_account)}"
```

Comment:
There is a terraform variable `support_email` that you could use instead of `@govpaas-alerting-%s@digital.cabinet-office.gov.uk`